### ISSUE TITLE
panic: could not process blockCopy from fork choice service: could not delete states prior to finalized check point

### ISSUE THREAD

**Issue by ethers** [prysmaticlabs/prysm#4119]
[2019-11-26 22:10:36]  INFO initial-sync: Processing block 108192/109712 - estimated time remaining 10m33s blocksPerSecond=2.4 peers=4/3
panic: could not process blockCopy from fork choice service: could not delete states prior to finalized check point, range: 108145, 108152: deleting states failed with 1 errors: could not delete genesis or finalized state

goroutine 217 [running]:
github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.(*InitialSync).Start(0xc0003c7bc0)
    beacon-chain/sync/initial-sync/service.go:132 +0xf1c
created by github.com/prysmaticlabs/prysm/shared.(*ServiceRegistry).StartAll
    shared/service_registry.go:44 +0x23e

**Comment by austinfromboston** [comment]
I am having a similar issue, beacon chain container is crashing during initial sync, started about a week ago.  If the issue isn't common, it's probably related to some bad local state from a prior panic.  

Started validating about 3 weeks ago, usually beacon or validator will panic after a couple of days, I'll update the container and resume, but this issue is blocking resume.  if I don't hear anything on this thread for a bit I'll wipe my local state and try rebuilding.

```
[2019-11-30 16:43:32]  INFO initial-sync: Processing block 108304/136877 - estimated time remaining 3h27m3s blocksPerSecond=2.3 peers=9/10
panic: could not process blockCopy from fork choice service: could not delete states prior to finalized check point, range: 108257, 108264: could not delete genesis or finalized state

goroutine 185 [running]:
github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.(*InitialSync).Start(0xc00049ba40)
	beacon-chain/sync/initial-sync/service.go:132 +0xf1c
created by github.com/prysmaticlabs/prysm/shared.(*ServiceRegistry).StartAll
	shared/service_registry.go:44 +0x23e
```

**Comment by rauljordan** [comment]
I believe this is resolved by @terencechain #3847, closing unless that is not the case
